server/integration_tests/standalone/README.md

```diff
@@ -0,0 +1,60 @@
+# Data Commons Standalone Integration Tests
+
+The tests in this folder are standalone tests that are run against the dev or prod website deployments.
```

Comment:
Thanks for the detailed README!  (More is great obv, the "norm" is typically a 2-line summary of what it does, and cmd lines on how to invoke)